app/services/customers/update_service.rb

```diff
@@ -6,6 +6,10 @@ def update(**args)
       customer = result.user.customers.find_by(id: args[:id])
       return result.not_found_failure!(resource: 'customer') unless customer
 
+      unless valid_metadata_count?(metadata: args[:metadata])
+        return result.not_allowed_failure!(code: 'invalid_number_of_metadata')
```

Comment:
Same comment regarding the type of error

Reply:
👍